.github/actions/jlab-a11y-test/action.yml

```diff
@@ -0,0 +1,116 @@
+name: Run accessibility regression tests on JupyterLab
+description: |
+  Builds JupyterLab from source, then runs accessibility regression tests
+  against it using Playwright. Outputs the results to a markdown file which is
+  then copied to the GitHub workflow run summary.
+inputs:
+  repository:
+    description: JupyterLab repository, for example gabalafou/jupyterlab
+  ref:
```

Comment:
The names of these inputs are copied from actions/checkout because they are passed directly through to that action.